src/main/java/com/programmers/springbootbasic/domain/voucher/FixedAmountVoucher.java

```diff
@@ -0,0 +1,37 @@
+package com.programmers.springbootbasic.domain.voucher;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class FixedAmountVoucher extends Voucher {
+    private final int amount;
+    private static final int MIN_AMOUNT = 10;
+    private static final int MAX_AMOUNT = 10_000_000;
```

Comment:
[nit]
```suggestion
    private static final int MIN_AMOUNT = 10;
    private static final int MAX_AMOUNT = 10_000_000;
    
    private final int amount;
```
개인적으로 상수, 불변 멤버변수, 변수 순서로  선은 하는 것을 선호합니다.

Reply:
`[5342e6c](https://github.com/prgrms-be-devcourse/springboot-basic/pull/725/commits/5342e6c313a4713573e43bb6e4d68b7ac581dcb9)`